sphinxcontrib/images.py

```diff
@@ -92,8 +95,8 @@ def directive_boolean(value):
     elif value.lower().strip() in ['no', '0', 0, 'false', 'none']:
         return False
     else:
-        raise ValueError(u"Please use on of: yes, true, no, false. "
-                         u"Do not use `{}` as boolean.".format(value))
+        raise ValueError("Please use on of: yes, true, no, false. "
```

Comment:
I'm not sure why it's here, that's nice support for both py2 and py3, no?

Reply:
I'm not sure I follow your comment. Why is what where? 

``s=u"Sömething ûnicode"`` is both valid in python3 and python2 afaik. In python2  s would have type ``unicode`` in python3 just ``str``.